docs/pages/blog/mui-core-v5.md

```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
 
 ### Core
 
-- Unstyled
-- Joy
+#### Unstyled components and hooks
+
+We still have a lot of work to do to have a great unstyled package.
+You can keep track of our progress in [#27170](https://github.com/mui-org/material-ui/issues/27170).
+
+Our high-level plan is to use the unstyled components and hooks as a base of Material components.
+We are aiming to complete this effort with the next major release (v6).
+As of now, you can evaluate the unstyled primitives in the `@mui/base` package or check out the new implementation of Material components in `@mui/material-next` package.
+Please note that both packages are in alpha and breaking changes can and will occur.
```

Comment:
I think that we can emphasize being in alpha so we can do breaking changes, it's likely already enough to be used in production. 

```suggestion
Please note that both packages are in an alpha state so we can release breaking changes.
```